php_phongo.c

```diff
@@ -144,38 +144,15 @@ zend_class_entry* phongo_exception_from_mongoc_domain(uint32_t /* mongoc_error_d
 			return php_phongo_connectiontimeoutexception_ce;
 		case MONGOC_ERROR_CLIENT_AUTHENTICATE:
 			return php_phongo_authenticationexception_ce;
-
 		case MONGOC_ERROR_COMMAND_INVALID_ARG:
 			return php_phongo_invalidargumentexception_ce;
-
-		case MONGOC_ERROR_STREAM_INVALID_TYPE:
-		case MONGOC_ERROR_STREAM_INVALID_STATE:
 		case MONGOC_ERROR_STREAM_NAME_RESOLUTION:
 		case MONGOC_ERROR_STREAM_CONNECT:
-		case MONGOC_ERROR_STREAM_NOT_ESTABLISHED:
```

Comment:
Apologies if I asked previously, but are the three `MONGOC_ERROR_STREAM` codes being removed here not longer used in libmongoc?

Does it make sense to remove all of these and instead add a case for the `MONGOC_ERROR_STREAM` domain below, where we return `php_phongo_connectionexception_ce`?

Reply:
Yes, the removed codes are no longer used in libmongoc, according to [Jesse's comment on a help ticket](https://jira.mongodb.org/browse/HELP-5525?focusedCommentId=1770461&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1770461) .